src/targets.rs

```diff
@@ -167,32 +167,32 @@ impl Target {
         };
 
         if config.base {
-            let _guard = TARGET_LOCK.write();
+            let _guard = TARGET_LOCK.write().unwrap_or_else(|e| e.into_inner());
```

Comment:
I suppose this technically could be left as-is (i.e. without the unwrap), but I'm not a fan of just ignoring `Result` types.